meetings/2023-09/september-26.md

```diff
@@ -0,0 +1,1974 @@
+# 26 September, 2023 Meeting Notes
+
+-----
+
+**Remote and in person attendees:**
+
+| Name                 | Abbreviation | Organization      |
+| -------------------- | ------------ | ----------------- |
+| Ujjwal Sharma        | USA          | Igalia            |
+| Michael Saboff       | MLS          | Apple             |
+| Christian Ulbrich    | CHU          | Zalari            |
+| Andrew Paprocki      | API          | Bloomberg         |
+| Rob Palmer           | RPR          | Bloomberg         |
+| Daniel Ehrenberg     | DE           | Bloomberg         |
+| Pablo Gorostiaga     | PGO          | Bloomberg         |
+| Bradford C. Smith    | BSH          | Google            |
+| Jack Works           | JWK          | Sujitech          |
+| Jordan Harband       | JHD          | Invited Expert    |
+| Daniel Minor         | DLM          | Mozilla           |
+| Eemeli Aro           | EAO          | Mozilla           |
+| Frank Yung-Fong Tang | FYT          | Google            |
+| Chip Morningstar     | CM           | Agoric            |
+| Waldemar Horwat      | WH           | Google            |
+| Ross Kirsling        | RKG          | Sony              |
+| Chris de Almeida     | CDA          | IBM               |
+| Devin Rousso         | DRO          | Invited Expert    |
+| Willian Martins      | WMS          | Netflix           |
+| Ben Allen            | BAN          | Igalia            |
+| Istvan Sebestyen     | IS           | Ecma              |
+| Chengzhong Wu        | CZW          | Alibaba           |
+| Samina Husain        | SHN          | Ecma              |
+| Richard Gibson       | RGN          | Agoric            |
+| Luca Casonato        | LCA          | Deno              |
+| Guy Bedford          | GB           | OpenJS Foundation |
+| Jesse Alama          | JMN          | Igalia            |
+| Shane F. Carr        | SFC          | Google            |
+| Ron Buckton          | RBN          | Microsoft         |
+| Kevin Gibbson        | KB           | F5                |
+
+??: If you do want to get, like, a plastic card, you can get them either at the Ginza or Ueno stations at the ticket office. You won’t be able to get them from the machines. But you show your passport, you’ll be able to get them at the ticket office.
+
```

Comment:
```suggestion
```

i thought i removed this from the google doc long before this PR went up; are we sure this was taken from the latest version of the google doc?

Reply:
I can confirm this is was taken from the lates version of google doc, dated 20 Oct.

Reply:
marking resolved, per @ljharb 's 👍 reaction above

Reply:
It should still be removed since it’s not actual meeting content.

Reply:
https://github.com/tc39/notes/pull/300
it has been removed.

Reply:
It has been removed. Thanks